src/styles/sidebar/components/search-input.scss

```diff
@@ -27,11 +27,8 @@
   width: 100%;
 
   // The search box expands when focused, via a change in the
-  // `max-width` property. In Safari, the <input> will not accept
```

Comment:
I read up on the original ticket, which is four years old. _It does not appear that this bug is still manifest in Safari_. This `0.1px` rule caused occasional layout issues in Chrome in certain circumstances, likely due to pixel rounding. You can see it here—note the one-pixel gap between the search icon and the one next to it:

<img width="478" alt="Screen Shot 2019-12-17 at 9 39 56 AM" src="https://user-images.githubusercontent.com/439947/71007684-68582d00-20b5-11ea-99a2-6b885245fbaa.png">